EmployeeDirectory.DAL/DataOperations/DeptProjectOperations.cs

```diff
@@ -0,0 +1,36 @@
+﻿using EmployeeDirectory.DAL.Contracts.Providers;
+using Microsoft.Data.SqlClient;
+
+namespace EmployeeDirectory.DAL.DataOperations
+{
+    public class DeptProjectOperations(IDBConnection dBConnection):IDeptProjectOperations
+    {
+        private readonly IDBConnection _dBConnection = dBConnection;
+
+        public Dictionary<string, string> GetDepartments(string tableName)
```

Comment:
Why you are taking "tableName" parameter. Since it is Department Repository the table name should be Department